src/parser/opcode.def

```diff
@@ -570,17 +570,5 @@ WABT_OPCODE(I64, I32, I64, I64, 4, 0xfe, 0x4e, I64AtomicRmw32CmpxchgU, "i64.atom
 /* Walrus interpreter only opcodes */
```

Comment:
Shouldn't this part be deleted?

Reply:
`Move32, Move64, Const32, Const64` are bytecodes of Walrus not wasm-instructions.
But these are also required in the parser, so I added it